components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -1,31 +1,54 @@
-import { Flex, Box, SimpleGrid } from "@chakra-ui/react";
+import {
+  Flex,
+  Text,
+  SimpleGrid,
+  Center,
+} from "@chakra-ui/react";
 import { modesOfTransport } from "../../utils/constants";
-import { TravelMethodButton } from "./TravelMethodButton";
+import CarpoolCounter from "./CarpoolCounter";
+import TravelMethodButton from "./TravelMethodButton";
 
-export function TravelMethodButtons({ handleTransPortMode }) {
+const ModesOfTransportButtons = ({
```

Comment:
let's remain consistent with our naming conventions, and stick with using 'Travel Method' instead of 'Mode Of Transport' 
```suggestion
const TravelMethodButtons = ({
```